letsencrypt/client/apache/configurator.py

```diff
@@ -75,27 +77,27 @@ class ApacheConfigurator(augeas_configurator.AugeasConfigurator):
     """
     zope.interface.implements(interfaces.IAuthenticator, interfaces.IInstaller)
 
-    def __init__(self, server_root=CONFIG.SERVER_ROOT, direc=None,
-                 ssl_options=CONFIG.OPTIONS_SSL_CONF, version=None):
+    def __init__(self, config, direc=None, version=None):
```

Comment:
Yes, you could probably have a single global config that would be retrieved with `zope.component.getUtility` (just like `IDisplay`). Unfortunately, there is not yet a clear line separating client functions from API functions in the code base, to be able to properly assess this now, but imagine API usage: singleton config might easily get into the user's way... Thoughts?